### PR TITLE
LiveViewState. TUI draw simplification. Node's data.

### PR DESCRIPTION
**Pull request by denisshevchenko**
`LiveViewState`. TUI draw simplification. Node's data.

### CODE REVIEW

File: app/LiveView.hs [hunk 1]
```diff
@@ -12,149 +13,171 @@ module LiveView (
...
-          deriving (Show, Eq)
+initLiveViewState
+    :: TopologyInfo
+    -> LiveViewState
+initLiveViewState (TopologyInfo nodeId _) = LiveViewState
+    { lvsQuit            = False
+    , lvsRelease         = "Shelley"   -- Should be taken from ..?
+    , lvsNodeId          = nodeIdNum
+    , lvsVersion         = showVersion version
+    , lvsCommit          = "bc39f7124" -- Should be taken from ..?
```
Comment (CodiePP) [inline app/LiveView.hs]: do the autogenerated data by Cabal contain such information?
https://www.haskell.org/cabal/users-guide/developing-packages.html#pkg-field-custom-setup-autogen-modules

Comment (CodiePP) [reply app/LiveView.hs]: I see we already have access to `Paths_cardano_node`

Comment (denisshevchenko) [reply app/LiveView.hs]: I see two autogenerated modules: `cabal_macros.h` and `Paths_cardano_node.hs`. But these modules contains different versions and system paths, it doesn't contain Git hash nor node's release..

File: app/LiveView.hs [hunk 2]
```diff
@@ -12,149 +13,171 @@ module LiveView (
...
+    , lvsMaxNetDelay     = 0
+    , lvsMempool         = 0
+    , lvsMempoolPerc     = 0
+    , lvsCPUUsagePerc    = 0
+    , lvsMemoryUsageCurr = 0
+    , lvsMemoryUsageMax  = 0
+    }
+  where
+    nodeIdNum = case nodeId of
+        CoreId num  -> num
```
Comment (CodiePP) [inline app/LiveView.hs]: could we make this a `nodeId :: Text`? and display for example `C3` or `R1` for core, relay, respectively.

**Pull request merged**